### PR TITLE
Graph view

### PR DESCRIPTION
**Pull request by alexwalterbos**
Added the library to make it easier to run.

### CODE REVIEW

**Comment by AlexKolpa** [comment]
:-1: Download [the zip](http://visjs.org/#download_install) and use that. This includes waaaaaay too much stuff that we don't need (you almost broke my browser trying to open the 'files changed' tab)

**Comment by alexwalterbos** [comment]
@AlexKolpa Whoa, didn't notice that was ~500 files. I've used the zip and removed a thing or two from that as well.

File: vis/docs/css/prettify.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
+.com {
```
Comment (AlexKolpa) [inline vis/docs/css/prettify.css]: docs file. Don't think we'll need this?

File: vis/docs/css/style.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
+html, body {
```
Comment (AlexKolpa) [inline vis/docs/css/style.css]: more docs

**Pull request closed without merging**